samples/client_query_w_positional_params.py

```diff
@@ -35,7 +35,9 @@ def client_query_w_positional_params() -> None:
             bigquery.ScalarQueryParameter(None, "INT64", 250),
         ]
     )
-    query_job = client.query(query, job_config=job_config)  # Make an API request.
+    query_job = client.query_and_wait(
```

Comment:
Please rename this variable, as the return value is no longer a QueryJob object. Instead it is a RowIterator object. "rows" or "results" could be a good name.

Reply:
Yes, sounds good. Will correct that now.